homeassistant/components/qnap/manifest.json

```diff
@@ -1,9 +1,12 @@
 {
   "domain": "qnap",
   "name": "QNAP",
+  "config_flow": true,
+  "dependencies": [],
   "codeowners": [],
```

Comment:
I suggest that you create a separate standalone PR, and add yourself as code owner.

Reply:
Opened PR #93915